appjs/views/application.js

```diff
@@ -21,6 +21,9 @@ var Application = BaseView.extend(require('./mixins/links.js'), {
     stack: { dir1: "up", dir2: "left", firstpos1: 25, firstpos2: 25 },
     buttons: { sticker: false }
   },
+  events: {
+    'click #savePreview': 'savePreview'
```

Comment:
Though hidden at the moment, without adding this, the save button located in the application navigation doesn't work (https://github.com/voxmedia/autotune/blob/save-notification/appjs/templates/application.ejs#L11-L12). Would certainly like to know if there is a better way to do this, though.